mimic/model/maas_objects.py

```diff
@@ -600,6 +603,55 @@ def get_host_info(self, available_types, requested_types, entity_id, clock):
                            current_timestamp)))
                      for rtype in requested_types])
 
+    def list_connections(self):
+        """
+        Lists connections that this agent has.
+        """
+        return [AgentConnection(id=self.id + '_' + dc,
+                                guid=self.connection_guid,
+                                agent_id=self.id,
+                                endpoint=dc,
+                                datacenter=dc)
+                for dc in ('dfw1', 'ord1', 'lon3')]
+
+
+@attributes([Attribute('id', instance_of=text_type),
```

Comment:
In the future, we should be moving to `attrs`, rather than `characteristic`.  There's nothing *wrong* with Characteristic, and it is still maintained, but attrs has a few subtle advantages and eventually we'd like to be on just one utility library of this type.  (You can really think of attrs as characteristic NG, it's a newer library by the same author for the same purpose.)